frontend/src/components/Select/Select.tsx

```diff
@@ -14,49 +15,31 @@ const { Option } = AntDesignSelect
 
 export interface OptionType {
 	value: string
-	displayValue: string | React.ReactNode
+	displayValue: React.ReactNode
 	disabled?: boolean
 	id: string
 	dropDownIcon?: React.ReactNode
 }
 
-type Props = Pick<
-	AntDesignSelectProps<any>,
-	| 'onChange'
-	| 'placeholder'
-	| 'loading'
-	| 'value'
-	| 'className'
-	| 'allowClear'
-	| 'notFoundContent'
-	| 'mode'
-	| 'dropdownRender'
-	| 'defaultValue'
-	| 'onSearch'
-	| 'children'
-	| 'optionLabelProp'
-	| 'filterOption'
-	| 'bordered'
-	| 'disabled'
-	| 'defaultActiveFirstOption'
-	| 'aria-label'
-	| 'tagRender'
-	| 'open'
-	| 'dropdownMatchSelectWidth'
-	| 'optionFilterProp'
-> & {
-	options?: OptionType[]
+type Props<
+	ValueType,
+	OptType extends BaseOptionType | DefaultOptionType,
+> = AntDesignSelectProps<ValueType, OptType> & {
+	options?: OptType[]
 	dropdownClassName?: string
 }
 
-const Select = ({
+const Select = <
```

Comment:
@ccschmitz fyi the antd select wasn't as flexible as I was hoping - its generic types don't really work the same way their signatures would suggest, e.g. the onChange callback is typed to receive an object of `ValueType` yet it actually receives objects with `label / value` fields